### ISSUE TITLE
How do you calculate the eigenvalue and eigenvector of COCO?

### ISSUE THREAD

**Issue by ducheng678** [princeton-vl/CornerNet-Lite#36]
I want to train my own dataset.
I see you use the eigenvalue and eigenvector during data sampling.
Could you tell me the method you use in details ?

**Comment by ducheng678** [comment]
Did you use the eigenvalue (eigenvectors) of covariance matrix?

**Comment by heilaw** [comment]
Yes, we randomly sample 5000 images from training set to calculate the covariance matrix.
